docker/scripts/startup-webserver.sh

```diff
@@ -10,5 +10,15 @@ sleep 2
 
 echo "[INFO] launching chromium instance"
 
+if [ -z $HOST ]; then
+	echo "[ERROR] Missing environment variable \$HOST."
+	exit 1
+fi
+
+if [ -z $PORT ]; then
+	echo "[ERROR] Missing environment variable \$PORT."
+	exit 1
+fi
+
 # Run python script on display 0
-DISPLAY=:99 python potoken-generator.py --bind 0.0.0.0
+DISPLAY=:99 python potoken-generator.py --bind $HOST --port $PORT
```

Comment:
There is no default value for $HOST. There is a need to have 0.0.0.0 as the host address.

Either you define this default value in the bash script or you pass it in the Dockerfile.

Reply:
Fixed via [commit 778648d](https://github.com/iv-org/youtube-trusted-session-generator/pull/23/commits/778648d1da83ffb5aa327a964950fd01471ea2bb)

Reply:
Can you instead make it simplier using this method: https://stackoverflow.com/questions/2013547/assigning-default-values-to-shell-variables-with-a-single-command-in-bash?

Reply:
Yes, I will.

However, isn't that a `bash` only feature? I thought `sh` doesn't support that.

Reply:
Ha yes. Feel free to switch the shabang to ash since this is based on alpine.

Reply:
Fixed with commits [95d0d90](https://github.com/iv-org/youtube-trusted-session-generator/pull/23/commits/95d0d90cf8e14075bafcda9b99f636c090ff250d) and [f8c1de3](https://github.com/iv-org/youtube-trusted-session-generator/pull/23/commits/f8c1de38131d9f90fd2527278874e0b37a149f45).